darwin/future/meta/objects/item.py

```diff
@@ -129,6 +130,31 @@ def set_layout(self, layout: ItemLayout) -> None:
         filters = {"item_ids": [str(self.id)]}
         set_item_layout(self.client, team_slug, dataset_id, layout, filters)
 
+    def assign(self, assignee_id: int, workflow_id: str | None = None) -> None:
+        if not assignee_id:
+            raise ValueError("Must specify assignee to assign items to")
```

Comment:
Urgh, I just merged this before I got email about your comments :( 
But this code is copied from other similar functions, so that keeps it consistent with them anyway.